.github/scripts/create_extension.sql

```diff
@@ -7,5 +7,6 @@ CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
 GRANT ALL ON SCHEMA sys to :user;
 ALTER USER :user CREATEDB;
 ALTER SYSTEM SET babelfishpg_tsql.database_name = :db;
+ALTER SYSTEM SET babelfishpg_tsql.migration_mode = :migration_mode;
```

Comment:
This was actually incorrect, there should be single quotes in this: `:'migration_mode'`

Reply:
Yeah, this is because your recent change has not yet been merged. Do I need to fix it in this PR?

Reply:
Ok, this change can with my PR